app/script/conversation/ConversationRepository.js

```diff
@@ -1849,7 +1849,7 @@ z.conversation.ConversationRepository = class ConversationRepository {
           throw error;
         }
 
-        amplify.publish(z.event.WebApp.CALL.STATE.DELETE, callMessageEntity.id);
+        amplify.publish(z.event.WebApp.CALL.STATE.DELETE, callMessageEntity.conversationId);
```

Comment:
On a side note, this should not be handled by an event. 
The promise chain passes by the `CallingRepo` so this guys is totally able to handle degradation error and end the call on his side.

Reply:
Agreed. All the calling state changes are historically event based though.